packages/manager/apps/enterprise-cloud-database/index.js

```diff
@@ -22,3 +22,8 @@ angular.module('enterpriseCloudDatabaseApp', [
   'ui.router',
   ovhManagerEnterpriseCloudDatabase,
 ]);
+// .config(
```

Comment:
Could be removed?

Reply:
Hello @antleblanc 👋
I put it in comment because this app don't use the Environment and I don't want to forget to restore the `setUserLocale` when we will update the app.
Should I remove it ? 🤔

Reply:
Then let's take it into account in separate pull request.